Content.Server/Atmos/Reactions/PlasmaFireReaction.cs

```diff
@@ -9,78 +10,81 @@ namespace Content.Server.Atmos.Reactions
     [DataDefinition]
     public sealed partial class PlasmaFireReaction : IGasReactionEffect
     {
-        public ReactionResult React(GasMixture mixture, IGasMixtureHolder? holder, AtmosphereSystem atmosphereSystem, float heatScale)
+        public ReactionResult React(GasMixture mixture,
+            IGasMixtureHolder? holder,
+            AtmosphereSystem atmosphereSystem,
+            float heatScale)
         {
-            var energyReleased = 0f;
-            var oldHeatCapacity = atmosphereSystem.GetHeatCapacity(mixture, true);
             var temperature = mixture.Temperature;
-            var location = holder as TileAtmosphere;
             mixture.ReactionResults[(byte)GasReaction.Fire] = 0;
 
-            // More plasma released at higher temperatures.
-            var temperatureScale = 0f;
```

Comment:
It would be more clear if you didn't change the variable names for a variable that shouldn't functionally change.

Reply:
To make sure I understand, "it would be more clear" refers to the change set rather than the actual implementation?

"Rate" doesn't seem like the right word to me since it's affecting both the amount of reactants and the relative proportion. So neither name seems best to me.
Would `temperatureFactor` or something similar be alright, or is leaving it as it was your preference?

Reply:
Try to put yourself in the shoes of a reviewer. I see a pretty dense and substantial diff and I need to prove to myself that there are no functional changes. Keeping variable names that haven't changed functionally would help with this.

Reply:
Believe me, I can empathize with a reviewer faced with dense reviews. My intention was to, admittedly, place a bit of one-time strain on reviewers of this PR to make the strain on maintainers of the code lighter. It's very much a subjective thing, so I've reverted the name of this local.